Formula/r/rpm.rb

```diff
@@ -70,6 +72,8 @@ def install
       -DWITH_SELINUX=OFF
       -DRPM_VENDOR=#{tap.user}
       -DENABLE_TESTSUITE=OFF
+      -DWITH_ACL=OFF
```

Comment:
Might be better at the top, but I can't suggest that

```suggestion
      -DCMAKE_INSTALL_RPATH=#{rpath}
      -DWITH_ACL=OFF
```